### PR TITLE
FiatTokenV1.1: Rescuable

### PR DESCRIPTION
**Pull request by petejkim**
Requires #296.

Relevant commits: 

* Implement Rescuable and rescueERC20 function
(https://github.com/centrehq/centre-tokens/commit/35d6fd4b04b415b7f62c772c44fe9d6211e3e71d)
* Run all V1 tests for V1.1(https://github.com/centrehq/centre-tokens/commit/8e52d223e9549e4f26ae7aa574a63a6d3f70c338)

### CODE REVIEW

**Comment by petejkim** [comment]
Rebased `modernize-codebase` branch

**Comment by petejkim** [comment]
Rebased from `master` branch.

File: test/helpers/storageSlots.behavior.ts [hunk 1]
```diff
@@ -5,33 +5,31 @@ import {
...
+      30e6,
+      10e6,
+    ];
+    const [
+      owner,
+      proxyAdmin,
+      masterMinter,
+      pauser,
+      blacklister,
+      minter,
```
[the remark this answers is not in the document]
Comment (petejkim) [reply test/helpers/storageSlots.behavior.ts:27]: Sure, adding it to this test will make it fail for FiatTokenV1 though. I can either put it somewhere else, or put it in this test and stop running tests for V1 post-V1.1.

[the remark this answers is not in the document]
Comment (eztierney) [reply test/helpers/storageSlots.behavior.ts:27]: Oh, thats true. Maybe make a new test in here and call it only from the v1_1 test? Maybe a `usesV11StorageSlots` in addition to the `useOriginalStorageSlotPositions`?

[the remark this answers is not in the document]
Comment (petejkim) [reply test/helpers/storageSlots.behavior.ts:33]: Added `version` param to storage slot test in https://github.com/centrehq/centre-tokens/pull/299/commits/00258ccf36e25bce6e0f62946d255d00160e3164

**Pull request merged**